shared/src/validation-schemas/podcast/create-podcast.validation-schema.ts

```diff
@@ -0,0 +1,10 @@
+import * as Joi from 'joi';
+import { PodcastCreatePayloadKey } from '~/common/enums/enums';
+import { PodcastCreatePayload } from '~/common/types/types';
+
+const createPodcastSchema = Joi.object<PodcastCreatePayload>({
```

Comment:
```suggestion
const podcast = Joi.object<PodcastCreatePayload>({
```

Check the A3 criteria